distributionviewer/core/static/css/logout.styl

```diff
@@ -1,7 +1,16 @@
 @import 'lib';
 
 .sign-out-wrapper {
+  .email {
```

Comment:
The className felt too generic but the project is small enough that yea we shouldn't have collisions. Good call.